### ISSUE TITLE
How abandoned is Bonfire?

### ISSUE THREAD

**Issue by dabockster** [ci-bonfire/Bonfire#1282]
Question in title. Since the support forums are gone and Github responses are few, is Bonfire still being actively maintained?

**Comment by lonnieezell** [comment]
Unfortunately, it's not actively maintained at the moment. Another guy was maintaining it for a couple of years, but then he bailed due to a job change, and I've too busy lately with CI4 to keep it actively maintained, especially since it really needs a major rewrite, IMHO. 

I've been considering posting on CodeIgniter forums to see if someone wants to take the project over. It's been great for a lot of people and I'd hate to see it completely die, but i just don't have the bandwidth at the moment. Know anyone? :)

**Comment by dabockster** [comment]
Unfortunately, I do not know anyone that would be willing to take it over. The only reason I'm using it myself is due to an older project that got pushed onto my work stack.

Anyways, thanks for the information. Hopefully the person currently paying me won't be too irked.

**Comment by Reconix** [comment]
Forgive my ignorance here but Sprint is the latest version, right ? and bonfire is the old one ?

**Comment by lonnieezell** [comment]
@Reconix not quite the same. Bonfire includes a GUI and a few different front-end things to make building an admin easier. I built Sprint to use a more modern coding style, Composer, etc, and be more like a super-charged CI, without the UI, but still with Auth, code generation, etc. They fit slightly different needs, and the intent was to rebuild Bonfire on top of Sprint, but I only got to use Sprint on one freelance project prior to taking a job about 1.5 years ago where I use Laravel all day, so it hasn't been maintained all that much since then, either, unfortunately. But, AFAIK, there's not a whole lot of people using Sprint. But who knows - I went 2-3 years thinking no one was really using Bonfire before finding out it had been used on hundreds of sites.
